hid-lg4ff.c

```diff
@@ -303,10 +312,26 @@ static const struct lg4ff_wheel_ident_info lg4ff_g29_ident_info2 = {
 	USB_DEVICE_ID_LOGITECH_G29_WHEEL
 };
 
+static const struct lg4ff_wheel_ident_info lg4ff_g923_ident_info = {
+	LG4FF_MODE_G923 | LG4FF_MODE_G29,
+	0xfff8,
+	0x1350,
+	USB_DEVICE_ID_LOGITECH_G923_WHEEL
+};
+
+static const struct lg4ff_wheel_ident_info lg4ff_g923_ident_info2 = {
```

Comment:
Same here but the third value should be the bcdDevice value after the wheel has switched to native mode.

Reply:
info2 deleted, because bcdDevice value does not change, stays 0x3800